src/pages/case-studies/ibm-commerce-platform.mdx

```diff
@@ -0,0 +1,281 @@
+used this project to strategically update their product to be able to better
+compete in the marketplace.
+
+## Proposed solution
+
+### Carbon transition
+
+The Carbon refresh would make the interface more clear, meaning users wouldn’t
+get lost in the flow and would be able to find what they needed. It also would
+align key pages before, during, and after the checkout process, creating a sense
+of continuity. The team hypothesized that as a result, users would be less
+likely to defect.
+
+The team mapped out their transition to Carbon, focusing at first on a core set
+of pages. While implementing Carbon components and aligning to the IBM Design
+Language, they took the opportunity to reevaluate their typography and
+illustrations. By improving their type hierarchy and illustrations they hoped to
+transform a somewhat dry experience to something delightful, while building more
+confidence and trust in the interface itself.
+
+<Row>
+<Column colLg={4}>
+
+![Order confirmation page prior to the redesign](images/checkout-before.png)
+
+<Caption>Order confirmation page prior to the redesign</Caption>
+
+</Column>
+
+<Column colLg={4}>
+
+![Order confirmation page after the redesign](images/checkout-after.png)
+
+<Caption>Order confirmation page after the redesign</Caption>
+
+</Column>
+</Row>
+
+### High value features
+
+The second part of the solution was identifying high impact features they could
+implement in tandem with the Carbon transition. Many of these features were
+self-contained and so they paired well with the overall transition to Carbon, as
+the team learned to design and develop with the design system.
+
+The global address form was one of these features. The new form validated
+addresses as users entered them, giving users confidence that they’d input the
+correct address. Additionally, the support team no longer had to manually follow
+up on invalid addresses, making the experience better for everyone.
+
+## The approach
+
+The transition to Carbon was an iterative process. Rather than updating the
+entire UI at one time, the Commerce Platform team transitioned their product
+slowly and tactfully. By adopting Carbon iteratively, they were able to see
+which changes made substantial impacts and use that to inform their roadmap and
+subsequent changes.
+
+### Educate on the design system
+
+Before jumping into adopting Carbon, it was important to spend time learning
+about and getting comfortable with Carbon. Shixie Shi Trofimov, the design lead
+of the transition project, led the developers through a crash course to align on
+definitions and terminology. She wanted designers and developers to speak the
+same language as they worked through a set of design specifications.
+
+<Row>
+<Column colLg={8}>
+
+![Carbon token quiz](images/token-quiz.png)
+
+<Caption>
+  Carbon token quiz that helped get developers comfortable with Carbon token
+  naming conventions.
```

Comment:
When do captions end in periods?

Reply:
They end in periods when they're full sentences, but I'll double check the captions with Jan